lib/matplotlib/pyplot.py

```diff
@@ -1185,6 +1185,11 @@ def close(fig: None | int | str | Figure | Literal["all"] = None) -> None:
         - ``str``: a figure name
         - 'all': all figures
 
+    Notes
+    -----
+    pyplot maintains a reference to figures created with `figure()` until they
+    are `close()`\ d.  Closing figures is thus necessary to prevent running out
```

Comment:
```suggestion
    pyplot maintains a reference to figures created with `figure()` until `plt.close()` is called 
    on them.  Closing figures is thus necessary to prevent running out
```

Reply:
sure